#____________________________________________________________________________________________
# findPickposx
#____________________________________________________________________________________________
# --

float64[] vertices
---
float64[] pick_posx
float64[] xyz
